tracer/src/Datadog.Trace/Configuration/ConfigurationKeys.Exporter.cs

```diff
@@ -80,5 +80,15 @@ internal static partial class ConfigurationKeys
         /// </summary>
         /// <seealso cref="ExporterSettings.PartialFlushMinSpans"/>
         public const string PartialFlushMinSpans = "DD_TRACE_PARTIAL_FLUSH_MIN_SPANS";
+
+        /// <summary>
+        /// Used to force the loader to start the tracer agent (in case automatic instrumentation is disabled)
+        /// </summary>
+        public const string AasEnableCustomTracing = "DD_AAS_ENABLE_CUSTOM_TRACING";
+
+        /// <summary>
+        /// Used to force the loader to start dogstatsd (in case automatic instrumentation is disabled)
+        /// </summary>
+        public const string AasEnableCustomMetrics = "DD_AAS_ENABLE_CUSTOM_METRICS";
```

Comment:
Should these go in `ConfigurationKeys.cs`? This file is for exporter settings only.

Reply:
Fixed in 18c25b94dbc2afc43c86450a638b85de37b6d58e